plugin/pkg/scheduler/scheduler.go

```diff
@@ -36,32 +37,6 @@ type Binder interface {
 	Bind(binding *api.Binding) error
 }
 
-// SystemModeler can help scheduler produce a model of the system that
```

Comment:
We probably have also some implementation of it that is no longer used. Can we remove it?